### PR TITLE
Companion PR for #8682

### PR DESCRIPTION
**Pull request by tomaka**
https://github.com/paritytech/substrate/pull/8682

### CODE REVIEW

File: node/network/bridge/src/lib.rs
Comment (kpp) [inline node/network/bridge/src/lib.rs:1290]:
```
    --> node/network/bridge/src/lib.rs:1290:5
     |
1290 |                 negotiated_fallback: None,
     |                 ^^^^^^^^^^^^^^^^^^^ `sc_network::Event::NotificationStreamOpened` does not have this field
```

https://gitlab.parity.io/parity/polkadot/-/jobs/911641#L398

Comment (kpp) [reply node/network/bridge/src/lib.rs:1290]: Looks like Cargo.lock should be updated

Comment (tomaka) [reply node/network/bridge/src/lib.rs:1290]: The Cargo.lock doesn't need to be updated for companion PRs. It's the Substrate CI (in https://github.com/paritytech/substrate/pull/8682) that automatically puts the fork of Substrate into Polkadot then builds it.
The companion PR, however, needs an approval.

**Review by kpp — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
@joao-paulo-parity simnet check is not required, but the companion merge has failed nevertheless, is this expected?

**Comment by TriplEight** [comment]
@ordian simnet check was supposed to be required (and was), but I had to unmark it bc Simnet was a bit broken. Now it's about to be stabilized and the "required" check will return.

**Comment by TriplEight** [comment]
But the fact that the bot has "required" checks configured elsewhere is rather odd, though.

**Comment by joao-paulo-parity** [comment]
> @joao-paulo-parity simnet check is not required, but the companion merge has failed nevertheless, is this expected?

The trigger-simnet job does not have `allow_failure: true`, so it is expected.

https://github.com/paritytech/polkadot/blob/46e39860719ed644a440ac2e776e14fe1d73c7ec/.gitlab-ci.yml#L366

**Comment by ordian** [comment]
> But the fact that the bot has "required" checks configured elsewhere is rather odd, though.

> The trigger-simnet job does not have allow_failure: true, so it is expected. That is solely what determines if a check can be skipped. We don't consider Github "required status" settings into account for skipping fallible statuses.

I don't understand what's the distinction between `allow_failure: true` and Github "requered status" setting, and will let you judge whether its needs to have `allow_failure: true`. But please make sure simnet check is ignored by bot until it's proved to be stable (e.g. not spuriously failing for a week).

**Comment by joao-paulo-parity** [comment]
@ordian 

https://github.com/paritytech/polkadot/pull/2958#issuecomment-834170649 is some sort of bug, don't mind it

> I don't understand what's the distinction between allow_failure: true and Github "requered status" setting

- `allow_failure: true` is the job's outcome from Gitlab.

- Github "required status" are statuses which will block the merge if they are not passing.

If any Gitlab check needs to be ignored temporarily, simply put `allow_failure: true` in .gitlab-ci.yml for that job. That setting is precisely for brittle/fallible/unstable jobs. It doesn't make sense to me to create custom rules in the bot when that setting is already proper.

**Pull request merged**
